tests/epyccel/modules/numpy_sign.py

```diff
@@ -0,0 +1,147 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+
+from pyccel.decorators import types
+
+def complex_pos():
+    import numpy as np
+    b = np.sign(complex(1+2j))
+    return b
+
+def complex_neg():
+    import numpy as np
+    b = np.sign(complex(-1-2j))
+    return b
+
+def complex64_pos():
+    import numpy as np
+    b = np.sign(np.complex64(64+64j))
+    return b
+
+def complex64_neg():
+    import numpy as np
+    b = np.sign(np.complex64(-64-64j))
+    return b
+
+def complex128_pos():
+    import numpy as np
+    b = np.sign(np.complex128(128+128j))
+    return b
+
+def complex128_neg():
+    import numpy as np
+    b = np.sign(np.complex128(-128-128j))
+    return b
+
+def int16_pos():
+    import numpy as np
+    b = np.sign(np.int16(16))
+    return b
+
+def int16_neg():
+    import numpy as np
+    b = np.sign(np.int16(-16))
+    return b
+
+def int32_pos():
+    import numpy as np
+    b = np.sign(np.int32(32))
+    return b
+
+def int32_neg():
+    import numpy as np
+    b = np.sign(np.int32(-32))
+    return b
+
+def int64_pos():
+    import numpy as np
+    b = np.sign(np.int64(64))
+    return b
+
+def int64_neg():
+    import numpy as np
+    b = np.sign(np.int64(-64))
+    return b
+
+def float_pos():
+    import numpy as np
+    b = np.sign(float(32.32))
+    return b
+
+def float_neg():
+    import numpy as np
+    b = np.sign(float(-32.32))
+    return b
+
+def float64_pos():
+    import numpy as np
+    b = np.sign(np.float64(64.64))
+    return b
+
+def float64_neg():
+    import numpy as np
+    b = np.sign(np.float64(-64.64))
+    return b
+
+def literal_complex_pos():
+    import numpy as np
+    b = np.sign(1+2j)
+    return b
+
+def literal_complex_neg():
+    import numpy as np
+    b = np.sign(-1-2j)
+    return b
+
+def literal_int_pos():
+    import numpy as np
+    b = np.sign(42)
+    return b
+
+def literal_int_neg():
+    import numpy as np
+    b = np.sign(-42)
+    return b
+
+def literal_float_pos():
+    import numpy as np
+    b = np.sign(42.42)
+    return b
+
+def literal_float_neg():
+    import numpy as np
+    b = np.sign(-42.42)
+    return b
+
```

Comment:
Please add tests for:
- argument is 0 :  e.g `np.sign(0)`/`np.sign(0.0)`
- complex argument with real part equal to 0 : e.g `np.sign(1j)`

Reply:
fixed here https://github.com/pyccel/pyccel/pull/1169/commits/3d87cc43c8f951a52ec7589eb230d3d605834360